Makefile

```diff
@@ -176,7 +176,10 @@ run-apk-tests:
 	exit $$_r
 
 run-performance-tests:
-	$(call MSBUILD_BINLOG,run-performance-tests,,Test) $(TEST_TARGETS) /t:RunPerformanceTests
+	_r=0 ; \
+	$(call MSBUILD_BINLOG,run-apk-tests,,Test) $(TEST_TARGETS) /t:RunApkTests /p:RunApkTestsTarget=RunPerformanceApkTests $(APK_TESTS_PROP) || _r=$$? ; \
```

Comment:
If we would only run performance apk tests, we might want to add the Mono.Android runtime test (Mono.Android_Tests) to them as well. To this item: https://github.com/xamarin/xamarin-android/blob/a63877371f942d1792f6de356443ce4c6db2bd6c/tests/RunApkTests.targets#L108